src/main/groovy/com/google/protobuf/gradle/ProtobufExtract.groovy

```diff
@@ -43,21 +49,29 @@ class ProtobufExtract extends DefaultTask {
    */
   private File destDir
   private Boolean isTest = null
+  private final ConfigurableFileCollection inputFiles = project.files()
 
   public void setIsTest(boolean isTest) {
     this.isTest = isTest
   }
 
+  @Input
   public boolean getIsTest() {
     Preconditions.checkNotNull(isTest)
     return isTest
   }
 
+  @InputFiles
+  @PathSensitive(PathSensitivity.NAME_ONLY)
+  public ConfigurableFileCollection getInputFiles() {
+    return inputFiles
+  }
+
   @TaskAction
   void extract() {
     destDir.mkdir()
     boolean warningLogged = false
-    inputs.files.each { file ->
+    inputFiles.each { file ->
```

Comment:
Does this break users of the plugin which register the input files with `protobufExtract.inputs.file(...)`? There doesn't seem to be another way how to register input files currently.

Reply:
It does, though I don't expect people to try to configure `ProtobufExtract` directly.